docs/src/running-tests-csharp.md

```diff
@@ -67,8 +67,30 @@ You can run a single test, a set of tests or all tests. Tests can be run on diff
 
 For more information see [selective unit tests](https://docs.microsoft.com/en-us/dotnet/core/testing/selective-unit-tests?pivots=mstest) in the Microsoft docs.
 
+## Debugging Tests
+
+Since Playwright runs in Node.js, you can debug it with your debugger of choice. Playwright comes with the Playwright Inspector which allows you to step through Playwright API calls, see their debug logs and explore [selectors](./selectors.md).
```

Comment:
```suggestion
Since Playwright runs in .NET, you can debug it with your debugger of choice in e.g. Visual Studio Code or Visual Studio. Playwright comes with the Playwright Inspector which allows you to step through Playwright API calls, see their debug logs and explore [selectors](./selectors.md).
```